Dlt 2319 update content doc page

jcarter-dp opened the pull request:
# PR Title

<!--- Feel free to remove any unused sections -->

## Obligatory GIF (super important!)

<!--- go to giphy.com, pick a gif, click share, copy gif link, paste within the () brackets below. -->

![](https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExOTBvcDJiY3BqM3JlNmRib3dlOHZuZ2t0dXE1ZTc2cTVvNGQwcmU3dyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/xVRjhzurfBoAaGvy4D/giphy.gif)


## :hammer_and_wrench: Type Of Change

<!--- Tick or place an `x` for the type of change. Should match the type in the commit message / PR title
in https://github.com/dialpad/dialtone/blob/staging/.github/COMMIT_CONVENTION.md -->

These types will increment the version number on release:

- [x] Fix
- [ ] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [x] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Jira Ticket

<!--- Enter the URL of the Jira ticket associated with this PR -->
[DLT-2319](https://dialpad.atlassian.net/browse/DLT-2319)

## :book: Description

<!--- Describe specifically what the changes are -->
Updated all markdown (h1, h2, h3, h4) headers to proper Title casing within the Grammar and Mechanics page of Dialtone's content guidelines.

## :bulb: Context
<!--- Describe the purpose of the changes -->
<!--- Why did we make these changes? -->
<!--- What problem(s) do they solve? -->

## :pencil: Checklist

<!--- Tick or place an `x` in all of the checkboxes that apply -->
<!--- Remove checkboxes that do not apply -->

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

For all Vue changes:

- [ ] I have added / updated unit tests.
- [ ] I have made my changes in Vue 2 and Vue 3. Note: you may sync your changes from Vue 2 to Vue 3 (or vice versa) using the `./scripts/dialtone-vue-sync.sh` script. Read docs here: [Dialtone Vue Sync Script](../packages/dialtone-vue3/.github/CONTRIBUTING.md#dialtone-vue-sync-script)
- [ ] I have validated components with a screen reader.
- [ ] I have validated components keyboard navigation.

For all CSS changes:

- [ ] I have used design tokens whenever possible.
- [ ] I have considered how this change will behave on different screen sizes.
- [ ] I have visually validated my change in light and dark mode.
- [ ] I have used gap or flexbox properties for layout instead of margin whenever possible.

If new component:

<!--- There are lots of things to remember when adding a new component to the system! This is so you don't forget any of them. -->

- I am exporting any new components or constants:
  - [ ] from the index.js in the component directory.
  - [ ] from the index.js in the root (either `packages/dialtone-vue2` or `packages/dialtone-vue3`).
- [ ] I have added the styles for the new component to the `packages/dialtone-css` package.
- [ ] I have created a page for the new component on the documentation site in `apps/dialtone-documentation`.
- [ ] I have added the new component to `common/components_list.js`
- [ ] I have created a component story in storybook
- [ ] I have created story / stories for any relevant component variants in storybook
- [ ] I have created a docs page for the component in storybook.
- [ ] I have checked that changing all props/slots via the UI in storybook works as expected.

## :crystal_ball: Next Steps

<!--- Describe any future changes that need to be made after merging the PR, especially any follow up tasks after release. --> Apply the same title casing throughout all other dialtone documentation pages.

## :camera: Screenshots / GIFs

<!--- Add these if necessary. Since we have deploy previews for every PR it may not always be. -->
<!--- Link any screenshots / GIFs below -->
![Screenshot 2025-02-11 at 2 47 33 PM](https://github.com/user-attachments/assets/55d8ee37-45b8-4d2d-910f-0d8aaa11247a)


## :link: Sources

<!--- Add any links to external reference material -->


[DLT-2319]: https://dialpad.atlassian.net/browse/DLT-2319?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

jcarter-dp committed:
Update the dialtone doc

Updating all the markdown headers within the "grammars and mechanics" content page of dialtone's documentation to have proper Title casing.

jcarter-dp committed:
Fix sentence vs title casing link

Under captialization section, fixed the link that properly aligns with sentence vs. title casing section.

francisrupert reviewed (commented):
Good initial start! A few things as you learn the way of things:

### PR Title Convention

Familiarize yourself with [Git Commit Convention](https://github.com/dialpad/dialtone/blob/staging/.github/COMMIT_CONVENTION.md). Look at [closed PRs](https://github.com/dialpad/dialtone/pulls?q=is%3Apr+is%3Aclosed) for examples. 

For this PR, this isn't a change that would increment the version number so I'd consider it a `chore`. So I'm thinking `chore: DLT-2319 update doc titles to title case`. I guess it's technically a "fix" of documentation, but I still see it as a basic chore that obviously shouldn't increment any version.

## Where to update

Short answer: all doc pages. 

Good start using `grammar-and-mechanics/index.md` as a starting point. Do the same for all doc pages that have a title of some sort, e.g. `h1`, `h2`, `etc`. 

Examples: each page within...

* `/components/` e.g. https://dialtone.dialpad.com/deploy-previews/pr-629/components/button.html
* `/design/` e.g. https://dialtone.dialpad.com/deploy-previews/pr-629/design/typography/
* `/utilities/` e.g. https://dialtone.dialpad.com/deploy-previews/pr-629/utilities/backgrounds/color.html
* `/guides/`, e.g. https://dialtone.dialpad.com/deploy-previews/pr-629/guides/contributing/
* `/about/` e.g. https://dialtone.dialpad.com/deploy-previews/pr-629/about/dialtone.html
* ...and so on. Basically all titles, everywhere. 
* Oh, and Homepage too, https://dialtone.dialpad.com/deploy-previews/pr-629/

jcarter-dp committed:
chore: DLT-2319 update doc titles to title case

Updating all doc pages within dialtone with markdown headers with proper title casing.

jcarter-dp committed:
Update word-wrap file

Update title casing in this file

francisrupert commented:
@jcarter-dp, 

Duplicate PR? Was this the initial one and then you created a different one at https://github.com/dialpad/dialtone/pull/637 ?

braddialpad commented:
@jcarter-dp this can be closed?
